### ISSUE TITLE
can't create hotspot

### ISSUE THREAD

**Issue by achou11** [digidem/edt-offline#27]
# Description

should be able to see the SSID for the offline EDT portal as an available network to connect to, but doesn't show up. Potentially relevant log:

```
Starting Earth Defenders Toolkit Hotspot and Captive-Portal
[network_manager::device:WARN] Undefined device type: 30
WiFi device: wlan0
[network_manager::dbus_api:ERROR] Get org.freedesktop.NetworkManager.AccessPoint::RsnFlags property failed on /org/freedesktop/NetworkManager/AccessPoint/48: wrong property type
Error: Getting access points failed
  caused by: D-Bus failure: Get org.freedesktop.NetworkManager.AccessPoint::RsnFlags property failed on /org/freedesktop/NetworkManager/AccessPoint/48: wrong property type
```

Potentially relevant issue: https://github.com/balena-os/wifi-connect/issues/416 with suggestion for fix

--- 

# Details:
  
Device Model: Raspberry Pi 4b
EDT Offline version: [v0.4.1](https://github.com/digidem/edt-offline/releases/tag/v0.4.1)

**Comment by Mattssmith94** [comment]
I'm also running into this same issue. I commented on https://github.com/balena-os/wifi-connect/issues/416, as their solution didn't work for me. Hopefully it gets resolved soon.

**Comment by luandro** [comment]
Changing WiFi stack to use https://github.com/balena-labs-research/python-wifi-connect

**Comment by luandro** [comment]
Should have been solved by #28
